nemo/collections/asr/modules/ssl_modules/augmentation.py

```diff
@@ -129,20 +128,15 @@ def __call__(self, batch: AudioNoiseBatch) -> AudioNoiseBatch:
             noise_signal = torch.zeros_like(audio_signal[i])
             noise_signal[mix_start_idx : mix_start_idx + mix_len] = mix_scale * noise_clip
 
-            # add noise to audio
-            noisy_audio[i] = audio_signal[i] + noise_signal
-            noisy_audio_len[i] = audio_lengths[i]
             noise[i] = noise_signal
             noise_len[i] = audio_lengths[i]
 
         return AudioNoiseBatch(
             sample_id=batch.sample_id,
             audio=batch.audio,
             audio_len=batch.audio_len,
-            noise=noise,
-            noise_len=noise_len,
```

Comment:
We want to keep `noise` and `noise_len` so that the caller can have full awareness of what noise is applied to the original audio, in case the noise is not simply added to the original audio.